### PR TITLE
[CIVP-15320] Add max_file_size param to civis_to_multifile_csv

### PR DESCRIPTION
**Pull request by uttercm**
**JIRA Ticket:**
[CIVP-15320](https://civisanalytics.atlassian.net/browse/CIVP-15320)

**Summary:**
Adding the max_file_size parameter to the civis.io.civis_to_multifile_csv

**Testing:**
- [x] Updated specs
- [x] Performed manual tests

Notes:
used the `civis.io.civis_to_multifile_csv` function to create files with different max file sizes and verified the file size
sql scripts are: [55697443](https://platform.civisanalytics.com/spa/#/scripts/sql/55697443) and [55697108](https://platform.civisanalytics.com/spa/#/scripts/sql/55697108)

**Additional Info:**

### CODE REVIEW

**Comment by uttercm** [comment]
@elsander is it okay that the build is failing due to the 3.4 error, I see that there is a PR to deprecate that so do I need to worry about that

**Comment by elsander** [comment]
We're supporting 3.4 until April, so it does need to pass on 3.4 for this upcoming release.

**Comment by uttercm** [comment]
see Jackson's comment in his PR: https://github.com/civisanalytics/civis-python/pull/341#issuecomment-571342110, seems like the failure on 3.4 needs has to do with PyYAML's requirements and not something we should do within this PR, if we want it to pass we will have to fix the dependancies

**Comment by jkulzick** [comment]
@elsander see my comment above. I guess @uttercm can pin to a lower version

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,7 +17,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - `ModelPipeline.register_pretrained_model` should persist the user-supplied
   estimator object indefinitely. (#331)
 - Fixed requirements.txt listing for `cloudpickle` -- `>=0.2`, not `<=0.2`. (#323)
-- Fixed issue in `civis.io.read_civis_sql` when returning data that contains 
+- Fixed issue in `civis.io.read_civis_sql` when returning data that contains
   double quotes. (#328)
```
Comment (jkulzick) [inline CHANGELOG.md]: Perhaps we should add an entry of about fixing the version of pyyaml for Python 3.4

**Pull request merged**